netbox_librenms_plugin/templates/netbox_librenms_plugin/htmx/device_validation_details.html

```
@@ -636,26 +627,302 @@ <h6 class="card-header d-flex justify-content-between align-items-center">
+                        <input class="form-check-input" type="radio" name="_promote_name_choice"
+                               id="promote-name-keep-{{ libre_device.device_id }}" value="keep" checked>
+                        <label class="form-check-label" for="promote-name-keep-{{ libre_device.device_id }}">
+                          Keep <code>{{ validation.existing_device.name }}</code>
+                        </label>
+                      </div>
+                      <div class="form-check">
+                        <input class="form-check-input" type="radio" name="_promote_name_choice"
+                               id="promote-name-new-{{ libre_device.device_id }}" value="new"
+                               data-override-target="override_name"
+                               data-override-value="{{ new_name }}"
+                               {% if not new_name or new_name == validation.existing_device.name %}disabled{% endif %}>
```

HTMLHint (1.9.2)

[error] Duplicate of attribute name [ new_name ] was found.

(attr-no-duplication)